openpype/hosts/fusion/plugins/create/create_exr_saver.py

```diff
@@ -47,3 +60,80 @@ def process(self):
             # Set file format attributes
             saver[file_format]["Depth"] = 1  # int8 | int16 | float32 | other
             saver[file_format]["SaveAlpha"] = 0
+
+        # Save all data in a "openpype.{key}" = value data
+        for key, value in instance_data.items():
+            saver.SetData("openpype.{}".format(key), value)
+
+    def collect_instances(self):
+
+        comp = get_current_comp()
+        tools = comp.GetToolList(False, "Saver").values()
+
+        # Allow regular non-managed savers to also be picked up
+        project = legacy_io.Session["AVALON_PROJECT"]
+        asset = legacy_io.Session["AVALON_ASSET"]
+        task = legacy_io.Session["AVALON_TASK"]
+
+        for tool in tools:
+
+            path = tool["Clip"][comp.TIME_UNDEFINED]
+            fname = os.path.basename(path)
+            fname, _ext = os.path.splitext(fname)
+            subset = fname.rstrip(".")
+
+            attrs = tool.GetAttrs()
+            passthrough = attrs["TOOLB_PassThrough"]
+            variant = subset[len("render"):]
+
+            # TODO: this should not be done this way - this should actually
+            #       get the data as stored on the tool explicitly (however)
+            #       that would disallow any 'regular saver' to be collected
+            #       unless the instance data is stored on it to begin with
+            instance = {
+                # Required data
+                "project": project,
+                "asset": asset,
+                "subset": subset,
+                "task": task,
+                "variant": variant,
+                "active": not passthrough,
+                "family": self.family,
+
+                # Fusion data
+                "tool_name": tool.Name
+            }
+
+            # Use the explicit data on the saver (if any)
+            data = tool.GetData("openpype")
+            if data:
+                instance.update(data)
+
+            # Add instance
+            created_instance = CreatedInstance.from_existing(instance, self)
+
+            # TODO: move this to lifetime data or alike
+            #       (Doing this before CreatedInstance.from_existing wouldn't
+            #        work because `tool` isn't JSON serializable)
+            created_instance["tool"] = tool
```

Comment:
There's been some discussion to potentially implement allowing non-serializable 'lifetime' data that is data for the instance that will remain as is for the current session - to e.g. allow storing a direct link the Fusion saver tool (= a node) that this instance was created for. That'd be a more direct correlation than storing `tool.Name` especially since Fusion can have multiple comps open and the logic always operating against the "current comp" otherwise you wouldn't know which 'comp' this tool belonged too, etc.